packages/phrases/src/locales/de/translation/admin-console/api-resource-details.ts

```diff
@@ -17,13 +17,13 @@ const api_resource_details = {
     create_subtitle: 'Define the permissions (scopes) needed by this API.', // UNTRANSLATED
     confirm_create: 'Create permission', // UNTRANSLATED
     name: 'Permission name', // UNTRANSLATED
-    name_placeholder: 'Read: Resources', // UNTRANSLATED
+    name_placeholder: 'Read:Resources', // UNTRANSLATED
     description: 'Description', // UNTRANSLATED
     description_placeholder: 'Able to read the resources', // UNTRANSLATED
     permission_created: 'The permission {{name}} has been successfully created', // UNTRANSLATED
     delete_description:
       'If this permission is deleted, the user who had this permission will lose the access granted by it.', // UNTRANSLATED
-    deleted: 'The permission {{name}} has been successfully deleted', // UNTRANSLATED
+    deleted: 'The permission "{{name}}" was successfully deleted!', // UNTRANSLATED
```

Comment:
Why? I think the previous one is better

Reply:
This PR is focused on the phrase update. As for the content, maybe discuss it with @fleuraly offline.

Reply:
ok, np. let's move on